Sources/Commands/SwiftTool.swift

```diff
@@ -566,6 +589,29 @@ public class SwiftTool<Options: ToolOptions> {
     func runLLBuild(manifest: AbsolutePath, llbuildTarget: String) throws {
         assert(localFileSystem.isFile(manifest), "llbuild manifest not present: \(manifest.asString)")
         
+        #if canImport(llbuildSwift)
+        if options.shouldEnableLLBuildLibrary {
+            try runLLBuildAsLibrary(manifest: manifest, llbuildTarget: llbuildTarget)
+            return
+        }
+        #endif
+        
+        try runLLBuildAsExecutable(manifest: manifest, llbuildTarget: llbuildTarget)
```

Comment:
Shouldn't this be inside the `canImport` above? It looks like this is defined on line `603` below.

Reply:
Don't think so. `runLLBuildAsExecutable` is the current default and needs to be called when no importing llbuild